### PR TITLE
Modal Rating Toast Grid done

### PR DESCRIPTION
**Pull request by Developers-codz**
Created Four P1 components
Modal Component
Rating component
Toast component
Grid component
Please do review and give your valuable feedback

### CODE REVIEW

File: css/style/modal.css [hunk 1]
```diff
@@ -0,0 +1,63 @@
+body{
+    position: relative;
+}
+.modal{
+    position:absolute;
+    z-index:30;
+    display:none;
+}
+.modal-lg{
+    width:400px;
```
Comment (arjun-computer-geek) [inline css/style/modal.css:10]: don't use px

File: css/style/modal.css [hunk 2]
```diff
@@ -0,0 +1,63 @@
+body{
+    position: relative;
+}
+.modal{
+    position:absolute;
+    z-index:30;
```
Comment (arjun-computer-geek) [inline css/style/modal.css]: is it need to give 30 value? Are you using a lots of stacking context?

File: docsite/components/toast.html [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+                <i class='fab fa-github fa-lg'></i>
+            </a>
+               </li>
+            <li class="inline-block padding-xs"><a class="light-text" href="https://www.linkedin.com/in/kajal-kumari-52bab41aa"><i class='fab fa-linkedin fa-lg'></i></a></li>
+            <li class="inline-block padding-xs"><a class="light-text" href="https://twitter.com/Kajal3310"><i class='fab fa-twitter fa-lg'></i></a></li>
+        </ul>
+        <small>© 2022 beadsUI <a class="decor-none light-text" href="https://github.com/developers-codz">Developers-codz</a> </small>
+        
+    </footer>
+    <script>
```
Comment (arjun-computer-geek) [inline docsite/components/toast.html]: I think you should use external js.

**Review by arjun-computer-geek — CHANGES_REQUESTED** [review]

File: docsite/components/toast.html (hunk 3 above)
Comment (Developers-codz) [reply docsite/components/toast.html]: As there was lots of file got created I put js there but yeah separated it. Thanks for reviewing.

File: css/style/modal.css (hunk 2 above)
Comment (Developers-codz) [reply css/style/modal.css]: corrected it

**Review by arjun-computer-geek — APPROVED** [review]
Yah now that looks good to me.

File: docsite/components/modal.html [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+                </div>
+                
+            </div>
+           <div class="section">
+            <h2 class="mb-lg" >Login modal</h2>
+            <p class="mb-lg font3 text-secondary">Login modals can be used for user login or signin.</p>
+            <div class=" centered shadow-box margin-md padding-y">
+               <button class="padding-sm reset rm-border bg-secondary light-text font3" id="modal-lg-popup-btn">Login</button>
+               <div class="modal modal-lg" id="login-modal">
+                <div class="modal-header reset " id="signup-close-btn">X</div>
```
Comment (vrinda-mahajan) [inline docsite/components/modal.html:85]: Just a suggestion, use icon instead of X.

File: css/style/toast.css [hunk 5]
```diff
@@ -0,0 +1,41 @@
+#toast {
+    visibility: hidden;
+    min-width: 250px;
+    margin-left: -125px;
```
Comment (vrinda-mahajan) [inline css/style/toast.css]: Avoid using px

**Review by vrinda-mahajan — CHANGES_REQUESTED** [review]

File: docsite/components/modal.html (hunk 4 above)
Comment (Developers-codz) [reply docsite/components/modal.html:85]: Ok I will take care of that from now

File: css/style/toast.css (hunk 5 above)
Comment (Developers-codz) [reply css/style/toast.css]: Thank you for pointing that out That was by mistake. Corrected it. :)

**Review by vrinda-mahajan — APPROVED** [review]
Good to merge.👍

**Pull request merged**